docs/references.bib

```diff
@@ -241,7 +241,8 @@ @Book{            borceux-vol3
 }
 
 @Book{            bosch-guntzer-remmert,
-  title         = {Non-Archimedean Analysis : A Systematic Approach to Rigid Analytic Geometry},
+  title         = {Non-Archimedean Analysis : A Systematic Approach to Rigid
+                  Analytic Geometry},
```

Comment:
I split these changes out to #16606; thanks for being diligent and running `bibtool` on your new entries!